### ISSUE TITLE
Styles!

### ISSUE THREAD

**Issue by iksaku** [LegendOfMCPE/HealthBar#2]
Hi and thanks for using HealthBar! Here you can leave your ideas for styles, here's an example of how can suggest you style:
`#CURRENTHEALTH#/#MAXHEALTH# (Ex. 10/20)`

Thanks, we will try hard to implement your ideas :D 

**Remember that you need to say exactly what do you want, and if it's possible, give us and image**
**If your idea is already implemented or it can't be added to Minecraft PE, it will be deleted.**

The styles list can be found [HERE!](https://github.com/LegendOfMCPE/HealthBar/wiki/Styles-list)

**Comment by gordonmhy** [comment]
Add ♥♥♥♥♥♥♥♥♥♥

**Comment by LukeDarling** [comment]
Unicode...

**Comment by gordonmhy** [comment]
'Above' and 'Under' is not usable because mcpe player's name tag allows 1 line only

**Comment by LukeDarling** [comment]
That is not true.

**Comment by iksaku** [comment]
@hoyinm yes it is: place the name above or under the Health using line breaks xD please test before commenting anything

**Comment by danield4n13l** [comment]
Pls,add this style!: 17x❤ or 17 ❤

**Comment by Danko24** [comment]
❤

**Comment by PEMapModder** [comment]
@iksaku why don't you do some PCRE replacement and allow more dynamic styles?

**Comment by austina5511** [comment]
Add health like black hearts plz even doe this is a really old plugin still works perfect best plugin yet :D

**Comment by MajorPlayz** [comment]
Include and option which disp,ays it in bars
|
For example, if ypu had full health you would have 20 bars ||||||||||||||||||||
This would be shortened on the actual tag due to fonts.

**Comment by MajorPlayz** [comment]
@austina5511 Had a great idea aswell

**Comment by quydrop** [comment]
$style = "§b⚜§c" . $health . "§e/§c" . $maxhealth . "§b⚜";
That's okey :D
